sql/2022/third-parties/usage_of_lite_youtube_embed.sql

```diff
@@ -0,0 +1,14 @@
+#standardSQL
+# Percent of pages using lite-youtube-embed
+
+SELECT
+  _TABLE_SUFFIX AS client,
+  COUNT(DISTINCT IF(app = 'lite-youtube-embed', url, NULL)) AS freq,
+  COUNT(DISTINCT url) AS total,
+  COUNT(DISTINCT IF(app = 'lite-youtube-embed', url, NULL)) / COUNT(DISTINCT url) AS pct
+FROM
+  `httparchive.technologies.2022_06_01_*`
+GROUP BY
+  client
+ORDER BY
+  client
```

Comment:
It's possible a page has no technologies detected by Wappalyzer so this would be more accurate as the following:

```suggestion
WITH totals AS (
  SELECT
    _TABLE_SUFFIX AS client,
    COUNT(DISTINCT url) AS total_pages
  FROM
    `httparchive.summary_pages.2022_06_01_*`
  GROUP BY
    client
),
youtube_embed AS (
SELECT
  _TABLE_SUFFIX AS client,
  COUNT(DISTINCT url) AS youtube_embed_pages
FROM
  `httparchive.technologies.2022_06_01_*`
WHERE
  app = 'lite-youtube-embed'
GROUP BY
  client
)

SELECT
  client,
  youtube_embed_pages,
  total_pages,
  youtube_embed_pages / total_pages AS pct_youtube_embed_pages
FROM
  totals
JOIN
  youtube_embed
USING (client)
ORDER BY
  client
```